examples/testapp/src/components/RpcMethods/shortcut/multipleChainShortcuts.ts

```diff
@@ -17,6 +17,13 @@ const walletSwitchEthereumChainShortcuts: ShortcutType[] = [
 			chain: soneiumMinato,
 		},
 	},
+		{
+		key: 'Ethereum',
+		data: {
+			chainId: '1',
```

Comment:
We could probably do this with the other chainIds too


```suggestion
			chainId: mainnet.id.toString(),
```

Reply:
fixed in https://github.com/StartaleGroup/app-sdk/pull/16/commits/8060f87c8b4a20880a621ffc642da9b12692978b